cmd/reject/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/canonical/specs-v2.canonical.com/config"
+	"github.com/canonical/specs-v2.canonical.com/db"
+	"github.com/canonical/specs-v2.canonical.com/google"
+	"github.com/canonical/specs-v2.canonical.com/specs"
+	"google.golang.org/api/drive/v3"
+)
+
+func main() {
+	var (
+		dryRun      bool
+		googleDocID string
+	)
+	flag.BoolVar(&dryRun, "dry-run", false, "perform a dry run without making actual changes")
+	flag.StringVar(&googleDocID, "google-doc-id", "", "reject a single doc (optional)")
+	flag.Parse()
+
+	c := config.MustLoadConfig()
```

Comment:
Let's standardize on `cfg` for variable names for configs here and elsewhere. `c` is a bit too vague.